packages/recomponents/src/plugins/r-toast-manager/r-toast-manager.js

```diff
@@ -2,7 +2,7 @@ import RToast from '../../components/r-toast/r-toast.vue';
 import RToastTypes from './r-toast-types';
 import DefaultErrorHandler from '../../common/helpers/default-error-handler';
 
-const ToastContainer = (Vue, globalOptions = {}) => ({
+const ToastManager = (Vue, globalOptions = {}) => ({
```

Comment:
probably `RToastManagerPlugin`?

Reply:
I think `RToastManager` is a more correct name, becasue this method gets used outside the plugin as well